### ISSUE TITLE
Labber python installation

### ISSUE THREAD

**Issue by tdegijsel** [Labber-software/Drivers#125]
Dear,

This is not a question regarding the drivers, but to the Labber python software installation. My python version and the Labber package seem to be inconsistent with each other. I have tried many ways of installing, but none of them works. What generally happens is that I cannot install the Labber package in a certain conda environment, but only globally. Because of that, my Spyder somehow does not open anymore. I prefer Spyder to Jupyter notebook, but Jupyter does open when the Labber package is installed. However, in Jupyter, the Labber package is never found. 

Basically, do you have any experience with this installation issue? Or do you now someone in the Labber organization who does? I need to have access to the Labber package to analyze the data from the Keysight Agilent Multimeter.

Best,
Thijs de Gijsel
